docs/reference/commandline/swarm_leave.md

```diff
@@ -55,4 +55,7 @@ To remove an inactive node, use the [`node rm`](node_rm.md) command instead.
 * [node rm](node_rm.md)
```

Comment:
I am not sure. It is added by mstanleyjones in the commit "Clarify usage of --force when used on a swarm manager".
@msabansal @thaJeztah PTAL
Thanks